modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first} after you configure your {rh-openstack} deployment.
+
+IMPORTANT: You cannot convert a dual-stack cluster into a single-stack IPv6 cluster. Additionally, deployments that rely on Fast Datapath are not supported.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing IPv6 subnet that is DHCPv6-stateful.
+* DNS is configured for the existing IPv6 subnet.
+* The IPv6 subnet is added to a {rh-openstack} router, and the router is configured to send router advertisements (RAs).
+* You added any additional IPv6 subnets that are used in the cluster to an {rh-openstack} router to enable router advertisements.
++
+NOTE: Using an IPv6 SLAAC subnet is not supported due to a known [OpenStack issue](https://bugzilla.redhat.com/show_bug.cgi?id=2304331) that prevents DNS from working.
```

Comment:
```suggestion
NOTE: Using an IPv6 SLAAC subnet is not supported as any defined `dns_nameservers` is not enforced by {rh-openstack} Neutron.
```
This is not really an issue, but it's the way it's expected to work according to the last comment on the bz.

Reply:
```suggestion
NOTE: Using an IPv6 SLAAC subnet is not supported as any defined `dns_nameservers` servers are not enforced by {rh-openstack} Neutron.
```